langchain/llms/llamacpp.py

```diff
@@ -230,6 +230,10 @@ def _call(
             return combined_text_output
         else:
             params = self._get_parameters(stop)
+
+            if params["max_tokens"] < 0:
```

Comment:
should handle case where max_tokens is None

Reply:
speaking of which, how does llama_cpp handle Nones for max_tokens? could it already be doing this inference under the hood?